app/screens/send-bitcoin-screen/send-bitcoin-confirmation-screen.tsx

```diff
@@ -219,7 +219,7 @@ const SendBitcoinConfirmationScreen: React.FC<Props> = ({ route }) => {
     destination,
     paymentType,
     sendingWalletDescriptor,
-    sendPayment: sendPaymentFn,
```

Comment:
I feel `sendPayment` is quite convoluted currently, this is probably the line that is most confusing to me.

we rename `sendPayment` to `sendPaymentFn`, then wrap into into a `useSendPayment()`, to get back a `sendPayment` again.

I feel renaming `sendPayment` in function that live outside the send bitcoin confirmation help clarify which function do what with more clarity. 

happy to use another name if you think there could be something better.